airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/component/TableSchemaEvolutionSuite.kt

```diff
@@ -309,58 +307,91 @@ interface TableSchemaEvolutionSuite {
         )
     }
 
-    fun `basic apply changeset`() {
-        `basic apply changeset`(
-            initialColumnNameMapping =
-                ColumnNameMapping(
-                    mapOf(
-                        "to_retain" to "to_retain",
-                        "to_change" to "to_change",
-                        "to_drop" to "to_drop",
-                    )
-                ),
-            modifiedColumnNameMapping =
-                ColumnNameMapping(
-                    mapOf(
-                        "to_retain" to "to_retain",
-                        "to_change" to "to_change",
-                        "to_add" to "to_add",
-                    )
-                ),
-        )
+    fun `apply changeset - append-append`() {
```

Comment:
nit: maybe something with more prose and descriptive like `applyChangeset handles append to append`